packages/frontend/components/SoundCard.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react'
+import styles from '../styles/SoundBoard.module.scss'
+
+let audio
```

Comment:
Hi this is a really good component !
It would be great if you can add the type for `audio` variable just to make sure everything is consistent across components and of couse use one of the advantage from typescript

Reply:
done